doc/neps/nep-0049.rst

```diff
@@ -315,7 +283,9 @@ mechanism. The PR was merged with no example code for using these features.
 Discussion
 ----------
 
-Not yet discussed on the mailing list.
+The discussion on the mailing list led to the ``PyDataMemAllocator`` struct
+with a ``context`` field like ``PyMemAllocatorEx`` but with a different
```

Comment:
Is intersphinx able to link to `PyMemAllocatorEx` here, if we do something like (but almost certainly not exactly) ``:c:type:`PyMemAllocatorEx` ``?

Reply:
fixed the intersphinx reference, your guess was correct